app/posts/apply-for-teacher-training/2021-02-08-work-history-iteration.md

```diff
@@ -0,0 +1,146 @@
+---
+title: Improving work history
+description: Asking for a full work history and simplifying the section by removing skills and experience.
+date: 2021-01-08
+related:
+  items:
+  - text: Work history breaks
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history-breaks/
+    description: Previous post about breaks in work history
+  - text: Work history - August iteration
+    href: https://bat-design-history.netlify.app/apply-for-teacher-training/work-history/
+    description: Previous change to the work history flow
+---
+Many providers told us that they want to see a candidate's full work history, not just the last 5 years - for safeguarding reasons.
+
+Candidates were spending a significant amount of time filling in the “Skills and experience relevant to teaching” question for each job, treating these as a “mini personal statement”. However, providers indicated that they didn’t always read these in detail.
```

Comment:
Providers don't read them BECAUSE they aren't always relevant to teaching. Worth saying I think.

Reply:
I'm not sure that necessarily the only reason - sometimes they might overlap with the personal statement, or just be too long-winded and vague. I'll see if I can find some more information from the original research conversations.